### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by aliali11121** [rainbow-me/rainbowkit#1540] [Feedback]
Hi! I can’t find the project’s discord (although as a user it’s important for me to be able to quickly communicate with the community to find out about technical problems or make suggestions on what can be improved).

**Comment by DanielSinclair** [comment]
We don't currently have a Discord or plans to host because of the unfortunately scam risk. But! The Wagmi team is currently working on launching a Discord, and access will be more widely available soon. This will be better suited for most questions related to Web3 development and any hiccups, as RainbowKit is built atop of Wagmi.
